scripts/xS2su09zcza9du09.js

```diff
@@ -1,8 +1,8 @@
-if (["Minor", "Moderate", "Major"].includes(this.item.system.specification.value))
+if ([game.i18n.localize("CORRUPTION.Minor"), game.i18n.localize("CORRUPTION.Moderate"), game.i18n.localize("CORRUPTION.Major")].includes(this.item.system.specification.value))
 {
 	return
 }
 
-let choice = await ItemDialog.create(ItemDialog.objectToArray({minor : "Minor", moderate : "Moderate", major : "Major"}, this.item.img), 1, "Choose Corruption Severity");
```

Comment:
You might as well include a translation for the "Choose Corruption Severity" text also

Reply:
I don't know how to add this request because I don't know how to use git very well, and I would like to submit another pull request when I have more time because I have other script text that I would like to have as a language file.